rpm/containers-common.spec

```diff
@@ -85,19 +85,21 @@ Summary: Extra dependencies for Podman and Buildah
 Requires: %{name} = %{epoch}:%{version}-%{release}
 Requires: container-network-stack
 Requires: oci-runtime
+Requires: iptables
```

Comment:
iptables dep should be dropped from RHEL/centos 10 as netavark must default to nftables. The iptables kernel modules were dropped anyway so it does not run correctly (https://issues.redhat.com/browse/RHEL-32374)

And thinking about this podman/containers-common does not use the nftables/iptables dep at all so shouldn't they be listed on netavark instead?

Reply:
Yes @Luap99 - it makes sense to maintain these types deps directly in the software requiring them. @lsm5 WDYT?

Reply:
I can drop nftables/iptables from the spec once a PR is merged to the netavark spec upstream first if you don't mind. I mean - I wouldn't block this PR for now but will do another one once netvark spec is updated upstream - if you are fine with that.

Reply:
btw. I just moved the iptables dep to Fedora section for now.

Reply:
Yeah I am fine to not move it in this PR.
But I don't see the fedora only change pushed here.

Reply:
My bad - just force pushed the change now.